mobify/platform-scaffold
# [Integration Manager] SFCC place order

This PR adds the submitPayment command to the SFCC connector. *Note* Currently we're able to create orders, but not get them to the confirmed status. 

 **JIRA**: https://mobify.atlassian.net/browse/WEBDATA-48
 **Linked PRs**: n/a

## Changes
- Add command to submitPayment on SFCC
- Add fetch Payment page command for SFCC
- Update createBasket util to return the full basket 

## How to test-drive this PR
- Run `npm run dev`
- Preview [the SFCC site](https://preview.mobify.com/?url=http%3A%2F%2Fmobify-tech-prtnr-na03-dw.demandware.net%2Fon%2Fdemandware.store%2FSites-2017refresh-Site%2Fdefault%2FHome-Show&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Add an item to your cart and proceed through the checkout to the payment page
- Enter correct information on the page and place the order
- You should be taken to the confirmation page
- You should see your new order in the SFCC admin site
